games-action/polymc/polymc-9999.ebuild

```diff
@@ -39,30 +39,41 @@ LICENSE="Apache-2.0 BSD BSD-2 GPL-2+ GPL-3 ISC LGPL-2.1+ LGPL-3+ MIT"
 
 SLOT="0"
 
-IUSE="debug lto test"
+IUSE="debug lto qt6 test"
 REQUIRED_USE="
 	lto? ( !debug )
 "
 
 RESTRICT="!test? ( test )"
 
-MIN_QT="5.12.0"
-QT_SLOT=5
+MIN_QT_5_VERSION="5.12.0"
+MIN_QT_6_VERSION="6.0.0"
 
 QT_DEPS="
-	>=dev-qt/qtconcurrent-${MIN_QT}:${QT_SLOT}
-	>=dev-qt/qtcore-${MIN_QT}:${QT_SLOT}
-	>=dev-qt/qtgui-${MIN_QT}:${QT_SLOT}
-	>=dev-qt/qtnetwork-${MIN_QT}:${QT_SLOT}
-	>=dev-qt/qttest-${MIN_QT}:${QT_SLOT}
-	>=dev-qt/qtwidgets-${MIN_QT}:${QT_SLOT}
-	>=dev-qt/qtxml-${MIN_QT}:${QT_SLOT}
+	!qt6? (
+		>=dev-qt/qtconcurrent-${MIN_QT_5_VERSION}:5
+		>=dev-qt/qtcore-${MIN_QT_5_VERSION}:5
+		>=dev-qt/qtgui-${MIN_QT_5_VERSION}:5
+		>=dev-qt/qtnetwork-${MIN_QT_5_VERSION}:5
+		>=dev-qt/qttest-${MIN_QT_5_VERSION}:5
+		>=dev-qt/qtwidgets-${MIN_QT_5_VERSION}:5
+		>=dev-qt/qtxml-${MIN_QT_5_VERSION}:5
+	)
+
+	qt6? (
+		>=dev-qt/qtbase-${MIN_QT_6_VERSION}:6[concurrent,gui,network,widgets]
+		>=dev-qt/qtsvg-${MIN_QT_6_VERSION}:6
+		>=dev-qt/qt5compat-${MIN_QT_6_VERSION}:6
+	)
 "
 
 # Required at both build-time and run-time
 COMMON_DEPENDS="
 	${QT_DEPS}
-	>=dev-libs/quazip-1.3:=[qt5(+)]
+
+	!qt6? ( >=dev-libs/quazip-1.3:=[qt5(+)] )
+	 qt6? ( >=dev-libs/quazip-1.3:=[qt6(+)] )
```

Comment:
This should be ˋ[qt6(-)]ˋ. The + indicates the flag should be treated as enabled when it is missing from the ebuild. This is what we want for qt5 because the ebuilds without qt* flags are all qt5 ebuilds. Qt6 support is only enabled in the ebuild that has a flag for it, so we should add a - here to indicate that the flag should be treated as disabled when it is missing.

Reply:
Oh, I see. I didn't know what it meant so I just copied the old one, sorry :sweat: